star-ros2/src/star_bringup/launch/slam.launch.py

```
@@ -164,6 +164,24 @@ def generate_launch_description():
 
+    # Foxglove Studio WebSocket bridge -- enables browser-based visualization
+    # at app.foxglove.dev without X11. Connects via ws://<PI5_IP>:8765.
+    # Install: sudo apt install ros-jazzy-foxglove-bridge
+    foxglove_bridge = Node(
+        package='foxglove_bridge',
+        executable='foxglove_bridge',
+        name='foxglove_bridge',
+        output='log',
+        parameters=[{
+            'port': 8765,
+            'address': '0.0.0.0',
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)